assets/queries/ansible/gcp/google_container_node_pool_auto_repair/query.rego

```diff
@@ -0,0 +1,51 @@
+package Cx
+
+CxPolicy [ result ] {
+  document := input.document[i]
+  tasks := getTasks(document)
+  task := tasks[t]
+  gcpContainer := task["google.cloud.gcp_container_node_pool"]
+  containerName := task.name
+	
+  object.get(gcpContainer, "management", "undefined") == "undefined"
+
+    result := {
+                "documentId":       input.document[i].id,
+                "searchKey":        sprintf("name={{%s}}.{{google.cloud.gcp_container_node_pool}}", [containerName]),
+                "issueType":        "MissingAttribute",
+                "keyExpectedValue": "google.cloud.gcp_container_node_pool.gcpContainer.management should be set to true",
```

Comment:
"keyExpectedValue": "google.cloud.gcp_container_node_pool.gcpContainer.management should be defined"